README.rst

```diff
@@ -50,45 +50,83 @@ How to run it
 
-- Edit the code where the db credentials are hardcoded (right now they are `zenodo:zenodo`, replace by your instances')
-- Run the migration ETL. In the existing case it requires JSONLines file with the records.
+- Install Invenio-RDM-Migrator, any other dependencies must be handled
+  in the Pipfile of your instance.
 
 .. code-block:: console
 
-    $ python run.py data/records-dump.jsonl
+    $ pip install invenio-rdm-migrator
 
-This will read from the jsonlines file, transform the entries to the RDM data model, output csv files with the data to load on the db (available in `/data/tables`), then perform a `COPY` operation to the DB. The created CSV files are suffixed with the timestamp, so there will be no clash between runs. However, if you wish to delete those files after running the migration, add the `--cleanup` flag to the run command from above.
+- Create/edit the configuration file on your instance, for example
+  `streams.yaml`:
 
-Implement your {Extract/Transform/Load}
-=======================================
+.. code-block:: yaml
 
-The package contains 4 folders and one main python file. The `data` folder is meant to host your data and the output of the process (e.g. `tables/`) and it is not version controlled.
+    records:
+        extract:
+            filename: /path/to/records.json
+        load:
+            db_uri: postgresql+psycopg2://inveniordm:inveniordm@localhost:5432/inveniordm
+            tmp_dir: /tmp/migrator
 
-The `run.py` file is where the ETL stream is defined and executed. You can hook in your custom extract, transform and load as long as they respect the contract for returning an iterator (so the `Stream.run` function works).
 
-Imagine you wish to read from an XML file and then transform it to RDM data model, so the load process stays the same.
+- You will need to create a small python script
+  putting together the different blocks of the ETL. You can find an eample
+  at `zenodo-rdm/site/zenodo_rdm/migrator/__main__.py`.
 
 .. code-block:: python
 
-    stream = Stream(
-        extract=XMLExtract(filename),
-        transform=XMLToRDMRecordTransform(),
-        load=PostgreSQLCopyLoad(),
-    )
+    from invenio_rdm_migrator.streams import StreamDefinition
+    from invenio_rdm_migrator.streams.records import RDMRecordCopyLoad
+
+    if \_\_name\_\_ == "\_\_main\_\_":
+        RecordStreamDefinition = StreamDefinition(
+            name="records",
+            extract_cls=JSONLExtract,
+            transform_cls=ZenodoToRDMRecordTransform,
+            load_cls=RDMRecordCopyLoad,
+        )
+
+        runner = Runner(
+            stream_definitions=[
+                RecordStreamDefinition,
+            ],
+            config_filepath="path/to/your/streams.yaml",
+        )
+        runner.run()
+
+- Finally, you can execute the above code. Since it is in the `__main__` file
+  of the python package, you can run it as a module:
+
+.. code-block:: console
+
+    $ python -m invenio_rdm_migrator
```

Comment:
```suggestion
    $ python -m my_site.migrator
```

minor: I would be more specific so that it's clear that this code goes in the users' instance.